bigquery/src/main/java/org/apache/zeppelin/bigquery/BigQueryInterpreter.java

```diff
@@ -90,9 +90,13 @@ public class BigQueryInterpreter extends Interpreter {
   static final String WAIT_TIME = "zeppelin.bigquery.wait_time";
   static final String MAX_ROWS = "zeppelin.bigquery.max_no_of_rows";
   static final String SQL_DIALECT = "zeppelin.bigquery.sql_dialect";
+  static final String REGION = "zeppelin.bigquery.region";
+
 
   private static String jobId = null;
   private static String projectId = null;
+  private static String location = null;
```

Comment:
I don't see any use for this new variable.

Reply:
Done.